src/docker-entrypoint.sh

```diff
@@ -1,172 +1,85 @@
 #!/bin/bash
 
 # Paranoia Level
-$(python <<EOF
-import re
-import os
-out=re.sub('(#SecAction[\S\s]{7}id:900000[\s\S]*tx\.paranoia_level=1\")','SecAction \\\\\n  \"id:900000, \\\\\n   phase:1, \\\\\n   nolog, \\\\\n   pass, \\\\\n   t:none, \\\\\n   setvar:tx.paranoia_level='+os.environ['PARANOIA']+'\"',open('/etc/modsecurity.d/owasp-crs/crs-setup.conf','r').read())
-open('/etc/modsecurity.d/owasp-crs/crs-setup.conf','w').write(out)
-EOF
-) && \
+sed -z -E -i 's/#SecAction.{7}id:900000.*tx\.paranoia_level=1\"/SecAction \\\n  \"id:900000, \\\n   phase:1, \\\n   nolog, \\\n   pass, \\\n   t:none, \\\n   setvar:tx.paranoia_level='"$PARANOIA"'\"/' /etc/modsecurity.d/owasp-crs/crs-setup.conf && \
```

Comment:
The `&&` concatenation of all the commands in this script is a bit fragile.

We should remove them all and add a `set -e` at the top of the script instead ([explanation](https://stackoverflow.com/questions/19622198/what-does-set-e-mean-in-a-bash-script)).